src/main/java/frc/trigon/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,166 @@
+
+public class ElevatorConstants {
+    private static final int
+            MASTER_MOTOR_ID = 13,
+            FOLLOWER_MOTOR_ID = 14;
+    private static final String
+            MASTER_MOTOR_NAME = "ElevatorMasterMotor",
+            FOLLOWER_MOTOR_NAME = "ElevatorFollowerMotor";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(MASTER_MOTOR_ID, MASTER_MOTOR_NAME),
+            FOLLOWER_MOTOR = new TalonFXMotor(FOLLOWER_MOTOR_ID, FOLLOWER_MOTOR_NAME);
+
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final InvertedValue
+            MASTER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            FOLLOWER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final boolean FOLLOWER_MOTOR_OPPOSES_MASTER = false;
+    private static final double
+            P = 110,
+            I = 0,
+            D = 0,
+            KS = 0,
+            KV = 0,
+            KG = 0,
+            KA = 0;
+    private static final GravityTypeValue GRAVITY_TYPE_VALUE = GravityTypeValue.Elevator_Static;
+    private static final StaticFeedforwardSignValue STATIC_FEEDFORWARD_SIGN_VALUE = StaticFeedforwardSignValue.UseClosedLoopSign;
+    private static final Rotation2d FORWARD_HARD_LIMIT_THRESHOLD = new Rotation2d(Angle.ofBaseUnits(500, Units.Degrees));
+    private static final Rotation2d REVERSE_HARD_LIMIT_THRESHOLD = new Rotation2d(Angle.ofBaseUnits(10, Units.Degrees));
+
+    private static final double GEAR_RATIO = 20;
+    static final double
+            MOTION_MAGIC_CRUISE_VELOCITY = 25,
+            MOTION_MAGIC_ACCELERATION = 25,
+            MOTION_MAGIC_JERK = MOTION_MAGIC_ACCELERATION * 10;
+
+    static final boolean FOC_ENABLED = true;
+
+    private static final double
+            MASS_KILOGRAMS = 5,
+            DRUM_RADIUS_METERS = 0.5,
+            MAXIMUM_HEIGHT_METERS = 1,
+            RETRACTED_ELEVATOR_LENGTH_METERS = 0.5;
+    private static final int MOTOR_AMOUNT = 2;
+    private static final DCMotor GEARBOX = DCMotor.getKrakenX60Foc(MOTOR_AMOUNT);
+    private static final boolean SHOULD_SIMULATE_GRAVITY = true;
+    private static final ElevatorSimulation SIMULATION = new ElevatorSimulation(GEARBOX,
+            GEAR_RATIO,
+            MASS_KILOGRAMS,
+            DRUM_RADIUS_METERS,
+            RETRACTED_ELEVATOR_LENGTH_METERS,
+            MAXIMUM_HEIGHT_METERS,
+            SHOULD_SIMULATE_GRAVITY
+    );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(0.25).per(Units.Seconds),
+            Units.Volts.of(2),
+            Units.Second.of(1000)
+    );
+
+    static final Pose3d ELEVATOR_ORIGIN_POINT = new Pose3d(1, 0, 1, new Rotation3d(edu.wpi.first.math.util.Units.degreesToRadians(10), 0, 0));
+    static final ElevatorMechanism2d MECHANISM = new ElevatorMechanism2d(
+            "ElevatorMechanism",
+            MAXIMUM_HEIGHT_METERS,
```

Comment:
this shouldn't be exactly the maximum height, add some more tolerance up, it'll be hard to see